php version.

digi3colin opened the pull request.

gaplo917 commented:
@digi3colin can you rename your files and split into two folder ?

Suggestion

```
php-9-for-loop/abcedfghppp.php
php-permutation/abcedfghppp.php
```

you can also think of a better name to describe instead of my suggestion :+1:

cherrry commented on php/index.php:
```diff
@@ -0,0 +1,56 @@
+}
+
+pure_brute_force();
+?>
```
http://stackoverflow.com/questions/4410704/why-would-one-omit-the-close-tag

digi3colin replied:
thanks cherrry's advice. The close tag omitted.

digi3colin committed:
added php & typescript code (+3 squashed commits)
Squashed commits:
[89c44e1] split solutions into 2 files.
[f9709a9] omit the close tag and split the file in 2 folder.
[eb9f391] add typescript-bruteforce code.

gaplo917 commented:
Please rebase & squash your commit into single commit  ( PHP related )

Similar case
#24

digi3colin commented:
is it helped?
The commits has been rebased.
Thanks for the advice!
